### ISSUE TITLE
Screen Reader does not announce the name information for edit box present under container box.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#204]
### User Experience:
If a screen reader will not announce the information about the edit box, then user will not be able to know the control functionality, and user enter incorrect value in the edit box, hence user is not getting the proper information about the control.

**Note:** User credentials should NOT be included in the bug.

### Repro-Steps:
- Navigate to the Left nav and select Pages button.
- Navigate to the Pages window. select API Reference option.
- Navigate to the any section details and select it
- Navigate to the Edit button and select it.
- Navigate to the "Section" blade and select Media button.
- Navigate to the various controls of the "select media" blade.
- Navigate to the various controls of the "Section" dialog box.
- Navigate to container box.
- Navigate to margin/padding edit box using keyboard.
- Verify whether screen Reader announces the full information for edit box or not.

### Actual Result:
Screen Reader does not announce the name information for edit box present under container box.

**Observation**
Screen reader announcing as "Hyphens" once the focus moves to the edit box under the container box.

### Expected Result:
Screen Reader should announce the name information for edit box present under container box.
Ex. Screen reader should announce the as "Margin Hyphens collapsed"
For padding: Screen reader should announce as "Padding Hyphens collapsed"

**Note:**
Once the user invokes the '-' edit box using keyboard, the pop is appeared, but Screen reader does not announce the state as collapsed/expanded

**Comment by msftedad** [comment]
Hi Team
Any update on this issue??
